CHANGES.rst

```diff
@@ -4,6 +4,7 @@ Changes
 Version 0.8.1 (UNRELEASED)
 ---------------------------
 
+- Adds support for kubernetes_job_timeout parameter.
```

Comment:
```suggestion
- Adds support for ``kubernetes_job_timeout`` parameter.
```

Reply:
Actually, better to have the same as other engines, for [example](https://github.com/reanahub/reana-workflow-engine-serial/pull/159/files#diff-ff3c479edefad986d2fe6fe7ead575a46b086e3bbcf0ccc86d85efc4a4c63c79R7).

```
Adds support for specifying ``kubernetes_job_timeout`` for Kubernetes compute backend jobs.
```